### DISCUSSION TITLE
Any hope for 3D Editor to manage scenes in UI?

### DISCUSSION THREAD

**Discussion by akfpnd** [google/filament#6324] [General]
I found your engine interesting to try out as for augmented reality mobile apps. But is a steep learning curve to pick it up.

To simplify entry level it could be nice to have visual editor to preview and edit scene in real time (in a way as Godot, Unity, Unreal does through code binding) in Android Studio or Xcode or AppCode (for both iOS and Android utilising Kotlin Multiplatform Mobile).

Any plans to develop such visual editor to increase adoption and usage Filament in real world applications?

**Comment by romainguy** [comment]
This is currently not planned. Doing this properly is a large amount of work we'd rather put into new features, optimizations, documentation, etc.

Note that since Filament supports glTF, any good glTF tool like Blender will go a long way to create complex scenes.
